pkg/migrations/migrations.go

```diff
@@ -280,6 +280,15 @@ func Migrate() {
 				return db.Where("site = ?", "WetVR").Delete(&models.Scene{}).Error
 			},
 		},
+		{
+			ID: "0014",
```

Comment:
can you tweak migration name to be in line with recent ones? (xxxx-brief-description)